test/test_proxy_tensor.py

```diff
@@ -1325,7 +1325,6 @@ def f(a, b, c, d, e):
     xfail('trapz', ''),  # aten.size.default - couldn't find symbolic meta function/decomposition
     xfail('trapezoid', ''),  # aten.size.default - couldn't find symbolic meta function/decomposition
     xfail('triangular_solve', ''),  # aten.triangular_solve.default - couldn't find symbolic meta function/decomposition
-    xfail('view_as_complex', ''),  # aten.view_as_complex.default - couldn't find symbolic meta function/decomposition
```

Comment:
hmm, how come there is no `view_as_real` here...

Reply:
yeah the allowed tests is set to torch.float so any op that doesn't currently support floating point type tensors is not currently tested by proxy_tensor tests